src/resolver/resolver.zig

```diff
@@ -2741,7 +2809,7 @@ pub const Resolver = struct {
         };
     }
 
-    pub fn loadAsIndex(r: *ThisResolver, dir_info: *DirInfo, path: string, extension_order: []const string) ?MatchResult {
```

Comment:
Does `loadAsIndex` continue to work when the `"browser"` field remaps? Did we always load from `dir_info.abs_path` before?

Reply:
I based it on this change: https://github.com/evanw/esbuild/commit/2051ab0b32466af34b1d0fbf8c6aa6e44c545a9d

I think what you're referring to is handled by the other `loadAsIndexWithBrowserRemapping` function, right? The current logic seems sound, but again I need to test this further.